proto/api.go

```diff
@@ -79,6 +79,7 @@ type Request interface {
 	gogoproto.Message
 	// Header returns the request header.
 	Header() *RequestHeader
+	KVHeader() *KVRequestHeader
```

Comment:
Also note that the `Sender` interface used to be named `KVSender`. We could bring that back (along with a `KVCall`) and separate the interfaces implemented by the different senders (requiring KVRequests in DistSender and LocalSender but any Request in HTTPSender).